### PR TITLE
Restore linter exclusions

### PR DESCRIPTION
**Pull request by marstr**

### CODE REVIEW

**Comment by marstr** [comment]
The following script was used to speed things up a bit:

``` bash
git diff --name-only azure-cli-2.0.67 upstream/dev -- src/command_modules/*/linter_exclusions.yml | xargs -I {} git checkout azure-cli-2.0.67 -- {}
```

File: .travis.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ jobs:
       env: PURPOSE='Dependency Check'
       script: ./scripts/dependency/check.sh
       python: 3.6
-#    - stage: verify
-#      env: PURPOSE='Lint Command Table and Help'
-#      script: ./scripts/ci/verify_linter.sh
-#      python: 3.6
```
Comment (tjprescott) [inline .travis.yml]: I would leave this disabled for now and just merge the relocation of the YML files

Comment (marstr) [reply .travis.yml]: Works for me!

Comment (marstr) [reply .travis.yml]: Fixed in f07fb88

**Pull request merged**